packages/editor/src/components/local-autosave-monitor/index.js

```diff
@@ -23,28 +23,27 @@ const requestIdleCallback = window.requestIdleCallback
 	: window.requestAnimationFrame;
 
 let hasStorageSupport;
-let uniqueId = 0;
 
 /**
  * Function which returns true if the current environment supports browser
  * sessionStorage, or false otherwise. The result of this function is cached and
  * reused in subsequent invocations.
  */
 const hasSessionStorageSupport = () => {
-	if ( typeof hasStorageSupport === 'undefined' ) {
-		try {
-			// Private Browsing in Safari 10 and earlier will throw an error when
-			// attempting to set into sessionStorage. The test here is intentional in
-			// causing a thrown error as condition bailing from local autosave.
-			window.sessionStorage.setItem( '__wpEditorTestSessionStorage', '' );
-			window.sessionStorage.removeItem( '__wpEditorTestSessionStorage' );
-			hasStorageSupport = true;
-		} catch ( error ) {
-			hasStorageSupport = false;
-		}
+	if ( hasStorageSupport !== undefined ) {
+		return hasStorageSupport;
 	}
 
-	return hasStorageSupport;
+	try {
+		// Private Browsing in Safari 10 and earlier will throw an error when
+		// attempting to set into sessionStorage. The test here is intentional in
+		// causing a thrown error as condition bailing from local autosave.
+		window.sessionStorage.setItem( '__wpEditorTestSessionStorage', '' );
+		window.sessionStorage.removeItem( '__wpEditorTestSessionStorage' );
+		hasStorageSupport = true;
+	} catch {
+		hasStorageSupport = false;
```

Comment:
Shouldn't we return the value in this `try/catch` block?

Reply:
I'd say we should return it after the `try / catch` block. Seems like in this branch we no longer return it at all in this function.

Reply:
Pushed a fix :+1: The `return` statement must have got lost during a `git add -p` when I wanted to commit the changes as two commits.